README.md

```diff
@@ -20,10 +20,11 @@ In tsconfig.json:
   "compilerOptions": {
     // ...
     "plugins": [
-      { 
-      	"transform": "@sapphi-red/ts-transformer-for-of", 
-      	"cacheLength": false  // cache the length of the array in `for-of` transformations. Defaults to false.
-      } 
+      {
+      	"transform": "@sapphi-red/ts-transformer-for-of",
+      	"cacheLength": false  // cache the length of the array in `for-of` transformations. Defaults to false.,
+        "exclusions": [] // exclude the transformations. The values of the array can be among: 'Array.for-of', 'Array.forEach', 'Array.map', 'Array.filter'
```

Comment:
- `Array.for-of`: `for-of-Array`
- `Array.forEach`/`Array.map`/`Array.filter`: `Array::forEach`/`Array.map`/`Array.filter`
I think this is better since they are not static methods.

Reply:
I changed everything to hyphen. 

https://github.com/sapphi-red/ts-transformer-for-of/pull/8/commits/3eda6d07c2054eaa8dca572b2ae8f1211286b5f7

I prefer these to be consistent with each other. We don't necessarily need to care about if the method is static, a prototype method, or a free for loop.

Reply:
Yes surely consistence is important. Thank you.